scripts/build_server

```diff
@@ -8,20 +8,20 @@ config="clang"
 while getopts "s:h" opt; do
   case "${opt}" in
     h)
-      echo -e "Usage: ${0} [-s dev|arm|asan|tsan]
+      echo -e "Usage: ${0} [-s x64|arm|asan|tsan]
 
 Build the specified version of the Oak Runtime server.
 
   -s    Server type:
-          - dev: development version of the server (default)
-          - arm: 'dev' server for ARM architecture
-          - asan: 'dev' server with address sanitizer
-          - tsan: 'dev' server with thread santizer
+          - x64: x64 version of the server (default)
```

Comment:
Not super keen on `x64` as the identifier for the normal version of the server &ndash; there's nothing architecture-specific being triggered here.  Maybe `base` or `core` or somesuch?

Reply:
Renamed to `base`